shell/browser/notifications/mac/cocoa_notification.mm

```diff
@@ -165,8 +166,10 @@
 void CocoaNotification::LogAction(const char* action) {
   if (getenv("ELECTRON_DEBUG_NOTIFICATIONS")) {
     NSString* identifier = [notification_ valueForKey:@"identifier"];
-    LOG(INFO) << "Notification " << action << " (" << [identifier UTF8String]
-              << ")";
+    if (identifier) {
```

Comment:
if there's no identifier on a notification this will crash on nullptr, so add a safety check